docker/linux-image.sh

```diff
@@ -424,7 +424,7 @@ EOF
 
     # need to reinstall the removed libgcc packages, which are required for apt
     if [[ "${arch}" == "${dpkg_arch}" ]]; then
-        apt-get install --no-install-recommends --assume-yes "${packages[@]}"
+        apt-get install --no-install-recommends --assume-yes "${libgcc_packages[@]}"
```

Comment:
I stumbled upon this... which I assumed to have been a typo since `packages` is likely not defined (its only used for wildcard kernel image searches).